packages/components/pages/f-mfa/src/components/Mfa.vue

```diff
@@ -43,11 +153,41 @@ export default {
 .c-mfa {
     display: flex;
     justify-content: center;
-    min-height: 80vh;
-    width: 80vw;
     margin: auto;
-    border: 1px solid red;
-    font-family: f.$font-family-base;
-    @include f.font-size(heading-m);
+    text-align: center;
+
+    @include f.media('>=narrow') {
+        width: 80vw;
+    }
+
+    h2 {
+        margin-top: 0;
+    }
+}
+
+.c-mfa-icon {
+    transform: translate(10%);
+    margin-top: - f.spacing(d);
+    width: 183px;
+}
+
+.c-mfa-card {
+    @include f.media('>=narrow') {
+        box-shadow: f.$elevation-01;
+    }
+}
+
+.c-mfa-formField {
+    text-align: left;
```

Comment:
Thin I see why you need this, and it comes from adding `text-align: center` to the `c-mfa`?

One potential fix for this would be for us to add `text-align: left;` to the `f-form-field` component input so that this doesn't get overriden by styles set higher up like this one (don't need to do as part of this PR, but would be good to do this).

Reply:
I've got another PR (#2063) to fix an issue with the visually required indicator, so I could try and include it in that.